.NET/Build.CI.cmd

```diff
@@ -26,6 +26,21 @@ if EXIST "%VSInstallDir%\MSBuild\!MsBuildVersion!\Bin\MSBuild.exe" (
 	EXIT /B
 )
 
+ECHO.
+ECHO # Check for empty and duplicate inputs in Specs
+Powershell -ExecutionPolicy Bypass "& {buildtools\checkSpec.ps1; exit $LastExitCode }"
+IF %ERRORLEVEL% NEQ 0 (
```

Comment:
This check and multiple code and messages could be eliminated if the .ps1 script already outputs the tight error message an a final error code.